particles/Tutorial/Javascript/3_RenderSlots/RenderSlots.arcs

```diff
@@ -6,18 +6,18 @@ particle ParentParticle in 'parent.js'
   // This particle renders to the root slot ("consumes" it), and provides a slot inside it called "mySlot" in which another particle can render
   // itself. The child particle will be rendered inside a special div with the identifier "mySlot", which this particle will need to provide in
   // its HTML.
-  consume root
-    provide mySlot
+  root: consumes Slot
+    mySlot: provides
 
 // The "child" particle. Instead of consuming "root" it consumes "mySlot", which connects it to the slot provided by ParentParticle.
 particle ChildParticle in 'child.js'
-  consume mySlot
+  mySlot: consumes
 
 recipe RenderSlotsRecipe
   ParentParticle
-    consume root
-      provide mySlot as slot0
+    root: consumes
```

Comment:
I propose `consumes *`, just `consumes` reads weird. I would also make `consumes *` required, not just optional. Same for handle fates, `use *`, `map *`. WDYT?

Reply:
`consumes Slot` is my preference for this. I would support requiring type annotations like this (using a type variable instead of * would be the escape hatch, which would aid in error message generation as there would not be a name for this and the user can slowly be more specific about which things should be the same type by reusing the same variable).

Reply:
Consumes * doesn't work. See #4043